geoh5py/shared/concatenation/concatenator.py

```diff
@@ -616,3 +618,237 @@ def update_array_attribute(
             self.data[alias] = values
 
         self.save_attribute(field)
+
+    def _no_value_property_group(
+        self, data_name: str | list[str]
+    ) -> tuple[dict, ConcatenatedPropertyGroup]:
+        """
+        Extract the data no data values and the common property group for a given data name.
+
+        :param data_name: The name of the data to extract.
+
+        :return: The data with the no data value and the property group
+        """
+        if isinstance(data_name, str):
```

Comment:
data_info dictionary is nice because it supports both depth and from-to in the same mechanic. 

We chat about selecting several data. I think it's a good idea, and we can select unique data as str.

Reply:
I create a sup-function to pass a str. Logics are separated